### DISCUSSION TITLE
Duda sobre TP1 - Lazos (Ej 10 - Factorial)

### DISCUSSION THREAD

**Discussion by alengeor** [INGCOM-UNRN-P1/cursada-2023#30] [Preguntas y respuestas] [answered]
Buenas gente!! Como andan?

Consulta, me agarró una laguna mental. ¿A que se refiere cuando dice "utilizando lazos"? Contexto:
![image](https://github.com/INGCOM-UNRN-P1/cursada-2023/assets/110181852/3f6278fe-1dca-42e4-9cc6-1aa298493488)

Según esta pagina web, la definición de lazo se refiere a un bucle: https://www.carlospes.com/minidiccionario/lazo.php

A ver a verrr

**Comment by LuchoGM** [comment, marked as answer]
Yo entendí lo mismo, utilice un bucle para realizar el programa.

**Reply by martinvilu** [reply]
Hay otra forma, que vamos a ver mas adelante en el cuatrimestre

**Comment by alengeor** [comment]
Pero... Que vendía a ser un bucle 🤔 
Un while... infinito¿?

*confusion sounds

**Reply by martinvilu** [reply]
Un sinónimo de lazo, y traducción de loop

**Reply by alengeor** [reply]
Va queriendo... Un loop no debe ser necesariamente infinito, o si? 🤔

**Reply by martinvilu** [reply]
No.

Tenés lazos definidos (`for`) y lazos indefinidos (`while`/`do..while`) (vamos a rever todo el lunes)

Ambos pueden ser infinitos, por las buenas, cuando uno quiere que sea de esta manera. O por las malas, cuando hay algún error en lo que escribimos y provoca que el programa no termine cuando esperamos.

**Reply by martinvilu** [reply]
Una observación menor @alengeor ;cuando la pregunta ya tiene su respuesta marcada, es mejor abrir un hilo nuevo, cosa de que la duda no quede sin ser contestada.

**Reply by alengeor** [reply]
Genial profe!! Lo tengo en cuenta para futuras consultas. Gracias!!
